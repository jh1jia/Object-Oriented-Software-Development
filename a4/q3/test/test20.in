test12.txt count z
test12.txt count z count 5
test12.txt count z count 0
test12.txt count z count z
