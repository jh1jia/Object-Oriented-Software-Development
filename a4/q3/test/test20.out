123456789101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869
12341678910111213141216171819202122232423262728293031323334343637383940414243444546474849607182931041112136147158169606162636461766676869
123456789111112131415161718192221222324252627282933313233343536373839444142434445464748495551525354555657585966616263646566676869
123456789101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869
